lib/modules/@apostrophecms/admin-bar/src/apos/components/TheApostropheAdminBar.vue

```diff
@@ -64,14 +64,17 @@ body {
   position: fixed;
   top: 0;
   left: 0;
-  height: 63px;
   width: 100%;
+  height: 63px;
   padding: 0 24px;
-  background: #1B232C;
-  font: 12px/1 'arial', sans-serif;
+  background: var(--a-background-inverted);
```

Comment:
Are we claiming "a" as a namespace for css variables? Does that impact project level? Any reason it's not apos?

Reply:
Yes, I think we wanted to keep it shorter.